ui_components/src/topbar/TopbarNotification.js

```diff
@@ -240,6 +240,32 @@ export function TopbarNotification({
                     </svg>
                 </a>
             }
+            {notification.type === "caravan" &&
+                <a href={notification.action_url}
+                    className={(notification.duration > 0) ? "topbar_notification_wrapper has_duration" : "topbar_notification_wrapper"}
+                    data-content={notification.message}
+                    data-time={timeRemainingDisplay}
+                >
+                    <svg className="topbar_notification_svg" width="40" height="40" viewBox="0 0 100 100">
+                        <polygon points="6,50 50,94 94,50 50,6" strokeWidth="8px" stroke="#5d5c4b" fill="#B09A65" />
+                        <polygon points="6,50 50,94 94,50 50,6" strokeWidth="2px" stroke="#000000" fill="#B09A65" />
+                    <image className="topbar_notification_icon" height="45" width="45" x="28%" y="30%" href="images/map/icons/caravan.png" />
+                    </svg>
+                </a>
```

Comment:
Stroke color should also be different than the other as well. A bunch of them are the same except for just image or stroke color. It might be better to create a general notification component that can be configured.